src/go/plugin/go.d/config/go.d/snmp.profiles/default/_aruba-switch-cpu-memory.yaml

```diff
@@ -28,10 +30,14 @@ metrics:
         # core check only
         name: memory.total
         scale_factor: 1000
+        description: total memory in kilobytes
+        unit: "kBy"
       - OID: 1.3.6.1.4.1.14823.2.2.1.2.1.15.1.3
         # core check only
         name: memory.used
         scale_factor: 1000
+        description: used memory in kilobytes
+        unit: "kBy"
```

Comment:
`kBy`, units it braces (e.g. `{packet}`). I don't think our UI supports (understands) auto-scaling for such units.

Reply:
@ralphm can you elaborate here? Do we prefer no auto-scaling for the units to favor the UCUM way of having them?

Reply:
@Ancairon, I assumed that these units do not work. Based on the fact that I have never seen them 😄

Reply:
> @Ancairon, I assumed that these units do not work. Based on the fact that I have never seen them 😄 

If they don't work, then this is a bug. We did work in the frontend as part of this effort: https://github.com/netdata/product/issues/3136.

Reply:
So @ralphm in this case should it be kBy or By?

Reply:
I don't know where `scale_factor` comes from. Do we use it? I know that DataDog uses this to consume the value in bytes instead of the original kilobytes that is provided by the source.  So if we do apply it, the unit should be `By`. If we don't `kBy`.

I am also wondering if `kilobyte` actually means 1000 bytes here, or really 1024 bytes. If the latter, the proper unit with prefix would be `kiBy`.

Reply:
(but I think SI-prefixes are more common in SNMP)